conf/drupal/config/acquia_connector.settings.yml

```diff
@@ -48,7 +48,7 @@ spi:
   site_name: REDACTED
   site_machine_name: REDACTED
   site_uuid: REDACTED
-  site_environment: prod
+  site_environment: null
```

Comment:
@arthurbaghdas Any idea why this value changed?

Reply:
Wrong export. Reverted